app/domain/operations/notices/ivl_oe_reverification_trigger.rb

```diff
@@ -0,0 +1,173 @@
+# frozen_string_literal: true
+
+require 'dry/monads'
+require 'dry/monads/do'
+
+module Operations
+  module Notices
+    # IVL open enrollment reverification notice
+    class IvlOeReverificationTrigger
+      include Dry::Monads[:result, :do]
+      include EventSource::Command
+      include EventSource::Logging
+
+      # @param [Family] family object (required)
+      # @return [Dry::Monads::Result]
+
+      def call(params)
+        _values = yield validate(params)
+        event_name = yield determine_event_name(params[:family])
+        payload = yield build_payload(params[:family])
+        event = yield build_event(payload, event_name)
+        result = yield publish_response(event)
+
+        Success(result)
+      end
+
+      private
+
+      def validate(params)
+        return Failure('Missing Family') if params[:family].blank?
+
+        Success(params)
+      end
+
+      def fetch_application(family)
+        applications = ::FinancialAssistance::Application.where(family_id: family.id, assistance_year: TimeKeeper.date_of_record.next_year.year)
+
+        determined_applications = applications.where(aasm_state: 'determined')
+        return determined_applications.max_by(&:created_at) if determined_applications.present?
+
+        applications.max_by(&:created_at)
+      end
+
+      # rubocop:disable Metrics/CyclomaticComplexity
```

Comment:
can you break this down to different methods instead of disabling rubocop error?